app/controllers/reviews_controller.rb

```diff
@@ -22,24 +22,22 @@ class ReviewsController < ApplicationController
   def index
     @title = t 'review.index_title'
     scope  = Review.list.
-      includes(:conclusion_final_review, :period, :tags, {
+      includes(
+        :conclusion_final_review, :period, :tags,
         plan_item: :business_unit,
         review_user_assignments: :user
-      }).
+      ).
       merge(ReviewUserAssignment.audit_team).
       references(:periods, :conclusion_final_review, :user)
 
     tagged_reviews = build_tag_search_for scope
 
-    build_search_conditions Review
-
-    reviews = @columns == ['tags'] ? scope.none : scope.where(@conditions)
-    order = @order_by || Review.default_order
+    reviews = search_params[:columns] == ['tags'] ? scope.none : scope.search(**search_params)
```

Comment:
Esto lo dejé igual que antes, pero en la vista se excluye el `tags`, hay un test nomás de esto, no se si lo inyectas por algún lado con algún flag o se puede volar. Además en el `COLUMNS_FOR_SEARCH` no está definido :man_shrugging:

Reply:
Dale, puedo combinarlo en el search derecho, pero a lo que voy es que se excluye el `tags` en la busqueda... por ende eso es algo que nunca va a hacer el usuario...

Lo "junto" y habilito el `tags` ?

Reply:
PD: Quizás la idea era deshabilitarlo solo para Oracle

Reply:
Es que se hacen dos búsquedas y después se juntan con el `or`. Si probás buscar por etiqueta funciona hermoso en la vista de los informes. Sea que seleccionás solo la columna de etiquetas o la combinás con otros. No recuerdo las específicas de la magia, el tema es que anda =)

Si para juntarlo tenés que habilitar el tags, que así sea, no hay un problema en eso, creo que venía más por el tema de como funcionaba el viejo `build_search`.

Si probá combinar búsquedas que incluyan un parámetro y una etiqueta, solo la etiqueta y eso. Esta parte si puedo dar _testimonio_ que la usan mucho.

Reply:
Vi tarde lo de Oracle, creería que no, eso debería funcionar sin más también ahí. Quizás no veo el elefante, pero de memoria entiendo que estaba habilitado.

Reply:
Jaja a lo que iba es que en `columns` NUNCA viene el tags, fuera de que busques con eso... Ya pruebo si lo puedo hacer andar.

En el index hay un hermoso, supongo que los joins/like se rompían de alguna forma. O quizás la separación solo la hiciste por oracle... vaya uno a saber.
```ruby
      <% if POSTGRESQL_ADAPTER  %>
        <%= make_filterable_column Tag.model_name.human(count: 0), { :class => 'd-none d-sm-table-cell' }, 'tags' %>
      <% else %>
        <%= make_not_available_column Tag.model_name.human(count: 0), { :class => 'd-none d-sm-table-cell' } %>
      <% end %>
```

Reply:
Que loco igual, cómo te digo: "no se como anda, pero anda" =) Ahí lo tenés, ese es mi argumento... jeje.

Sobre el `if` en la vista, si está entonces dejalo así. Seguro hay dolor en el camino, y no vale ni la pena tratar de implementarlo para Oracle.

Reply:
Encontré el problema jaja, con 1 solo argumento anda perfecto o con `OR`s pero si le metes por ej `AS-AC y AS-AF` el sql hace `name like 'AS-AC' AND name like 'AS-AF'` y obviamente esa condición no se cumple nunca...  Por lo cual habría que (en el caso de los tags) cambiar todos los AND por los OR's...  voy a ver como me va con esto jaja. Sino junto los 2 `where` y a la goma.

Seguiremos informando...

Reply:
Al final terminé combinando los 2 (lo que estaba antes con lo que quería hacer yo). Quedo bastante decente, el test que cree está medio a ojo, si tenes algo mas pulenta o lo ves "mal"  avisame. 

Ya está listo para review :+1: